Bug 659 : Storybook > Components > Contact Us > Default > Displaying Multiple Error Messages At the One Time

harshitsaini-waiin opened the pull request:
## Description

Fixed multiple error messages being displayed at one time in the Contact Us default component. 

Fixes:
- Added changes to fix the multiple error messages being displayed.
- Added the validations for invalid email in the Input element to handle the alignment issue.
- Corrected the "Message is Invalid" error message to "Message is required"
- Added '*' to the Message label as it is a required field.

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Screenshots
![image](https://github.com/user-attachments/assets/e9177a5b-6613-4677-9991-114e6f2b1952)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

harshitsaini-waiin committed:
Commit changes for bug 659

Fixed multiple message display issue in Contact Us component

Ketan-Khalate commented on raaghu-elements/src/rds-input/rds-input.tsx:
```diff
@@ -253,13 +268,22 @@ const RdsInput = React.forwardRef<HTMLInputElement, RdsInputProps>((props, ref)
-                        )}
-                    </div>
-                )}
+                <div className="form-control-feedback d-block text-end">
```
@harshitsaini-waiin we don't pass validation message directly from element as we are handling error message with respect to label so remove Email div and handle it from component itself

harshitsaini-waiin replied:
@Ketan-Khalate resolved above comments.
Handled validations from component and removed validations which were passed directly from element.

harshitsaini-waiin committed:
Commit changes for resolving comments

Handled validations from component, removed validation messages from input element

harshitsaini-waiin committed:
Merge branch 'development' into users/harshit/bug/659/multiple-error-messages-displayed

Ketan-Khalate committed:
Merge branch 'development' into users/harshit/bug/659/multiple-error-messages-displayed

Ketan-Khalate commented on raaghu-components/src/rds-comp-contact-us/rds-comp-contact-us.tsx:
```diff
@@ -14,8 +14,7 @@ const RdsCompContactUs = (props: RdsCompContactUsProps) => {
     const [error2, setError2] = useState("");
```
@harshitsaini-waiin give proper name instead error1, error2 and error3 give it as per field name

harshitsaini-waiin committed:
Commit changes for resolving comments

Ketan-Khalate committed:
Merge branch 'development' into users/harshit/bug/659/multiple-error-messages-displayed

Ketan-Khalate commented:
Changes Reviewed and PR Approved
